src/distrs/binom.jl

```diff
@@ -1,19 +1,25 @@
 # functions related to binomial distribution
 
 import .RFunctions:
-    binompdf,
-    binomlogpdf,
-    binomcdf,
-    binomccdf,
-    binomlogcdf,
-    binomlogccdf,
+    # binompdf,
+    # binomlogpdf,
+    # binomcdf,
+    # binomccdf,
+    # binomlogcdf,
+    # binomlogccdf,
     binominvcdf,
     binominvccdf,
     binominvlogcdf,
     binominvlogccdf
 
-# pdf for numbers with generic types
-binompdf(n::Real, p::Real, k::Real) = exp(binomlogpdf(n, p, k))
+binompdf(n::Real, p::Real, k::Real) = betapdf(k + 1, n - k + 1, p)/(n + 1)
 
-# logpdf for numbers with generic types
-binomlogpdf(n::Real, p::Real, k::Real) = -log1p(n) - logbeta(n - k + 1, k + 1) + k * log(p) + (n - k) * log1p(-p)
+binomlogpdf(n::Real, p::Real, k::Real) = betalogpdf(k + 1, n - k + 1, p) - log(n + 1)
```

Comment:
I guess `log1p(n)` and is more expensive but not more accurate than `log(n + 1)` for `n >> 1`?

Reply:
That was my thinking. `log1p` is only relevant for small arguments and that won't happen here.